code/datums/components/riding/riding_vehicle.dm

```diff
@@ -49,7 +47,6 @@
 			vehicle_parent.unbuckle_mob(user, TRUE)
 			user.visible_message("<span class='danger'>[user] falls off \the [vehicle_parent].</span>",\
 			"<span class='danger'>You fall off \the [vehicle_parent] while trying to operate it without being able to hold on!</span>")
-			user.Stun(3 SECONDS)
```

Comment:
why removing?

Reply:
The stun applies also when you get thrown off the bike and since its not a nochain you can get stunned for multiple seconds if you get stunned by any xeno ability